packages/gasket-plugin-https/package.json

```diff
@@ -3,6 +3,11 @@
   "version": "6.5.1",
   "description": "Create http/s servers with graceful termination",
   "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
```

Comment:
Just wondering, why is this the only package.json that needs the `files` property?

Reply:
I added here because this package is missing an `.npmignore` so it's currently publishing unnecessary files. Added `files` in here to fix rather than `.npmignore` since IMO that's a far superior approach that requires less upkeep.

Reply:
Got it - yes `files` is my preferred approach as well.